### PR TITLE
Add new OIL License

### LINKED ISSUE #2384: Add OIL licenses
See https://github.com/StarkDrones/OPNLv1/blob/main/.github/LICENSE.md and  https://github.com/StarkDrones/OPNLv2/blob/main/.github/LICENSE.md

There seems to some use in the wild https://github.com/search?q="Open+Innovation+License"&type=code that may jsutify a named license.
The alternative is to create multiple detection rules instead.
The licenses are somehow problematic for reuse as they are Company-specific

### PR DESCRIPTION
**Pull request by akugarg**
Signed-off-by: Akanksha Garg akanksha.garg2k@gmail.com
Fixes #2384 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: src/licensedcode/data/licenses/oil-1.0.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+key: oil-1.0
+short_name: OIL-1.0
+name: Open Innovation License version 1.0
+category: Permissive
+owner: Stark Drones Corporation
+notes: From https://github.com/StarkDrones/OPNLv1/blob/main/.github/LICENSE.md
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/oil-1.0.yml]:
```suggestion
other_urls:
    - https://github.com/StarkDrones/OPNLv1/blob/main/.github/LICENSE.md
```

File: src/licensedcode/data/licenses/oil-1.0.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+homepage_url: https://www.en.patent.aau.dk/open-innovation-licensing/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/oil-1.0.yml]: I do not think this is the home page and that this Danish outfit is related to this license here. They vaguely use similar names. Or may be I am missing something?

File: src/licensedcode/data/licenses/oil-2.0.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+key: oil-2.0
+short_name: OIL-2.0
+name: Open Innovation License version 2.0
+category: Permissive
+owner: Stark Drones Corporation
+notes: From https://github.com/StarkDrones/OPNLv2/blob/main/.github/LICENSE.md
+homepage_url: https://www.en.patent.aau.dk/open-innovation-licensing/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/oil-2.0.yml]: Same comment as above for notes and homepage

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks! 
I have a few comments for your consideration.

IMHO the correct text_url would be this:

- other_urls could have something such as 
  - https://github.com/StarkDrones/OPNLv2
  - https://github.com/StarkDrones/OPNLv1
- homepage_url should be https://github.com/StarkDrones/OPNL
- The license key should be `opnl-1.0` and `opnl-2.0` respectively as their authors name them as `OPNL` not `OIL`

File: src/licensedcode/data/licenses/oil-2.0.yml (hunk 3 above)
Comment (akugarg) [reply src/licensedcode/data/licenses/oil-2.0.yml]: Hey! @pombredanne  I have made the requested changes . Please review !

**Review by steven-esser — CHANGES_REQUESTED** [review]
@akugarg Can you rebase this into a single commit?

**Comment by pombredanne** [comment]
@akugarg LGTM! :+1:  we just need a rebase before we can merge:)

**Review by pombredanne — APPROVED** [review]
LGTM! Thank you++
Merging!

**Pull request merged**